src/main/java/org/sunbird/bpreports/postgres/repository/WfStatusEntityRepository.java

```diff
@@ -0,0 +1,13 @@
+package org.sunbird.bpreports.postgres.repository;
+
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+import org.sunbird.bpreports.postgres.entity.WfStatusEntity;
+
+import java.util.List;
+
+public interface WfStatusEntityRepository extends JpaRepository<WfStatusEntity, String>{
+
+    List<WfStatusEntity> getByApplicationId(String applicationId);
```

Comment:
applicationId -- would be userId value... 
so let's use userId -- since we are planning to remove this applicationId column in the next version.